db/migrate/20230719070614_create_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users, &:timestamps
+  end
```

Comment:
- [ ] It's fine implementing the users table, but kindly ensure that all columns as stated in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) are added appropriately. ℹ️ 

```suggestion
   create_table :users do |t|
    t.string :name
    t.string :photo 
    t.text :bio
    t.integer :posts_counter
    t.timestamps 
  end
```